docs/content/tools/xdebug.md

```diff
@@ -188,5 +188,21 @@ To debug Drush commands using Xdebug and VSCode, add the following to your path
     - Search for "php-debug"
     - Click "Settings" button below plugin
     - "Server" can be set to `*`
-    - "Server Listen Port" should be set to 9000
+    - "Server Listen Port" should be set to 9003
     - Make sure "Continue to listen for debug sessions even if the debugger windows are all closed" is checked. This will make the debugger window open automatically.
+
+## Configuring Prior Versions
+
+For versions of XDebug prior to v3.0.0, the following changes will need to be made to the projects `.docksal/docksal.yml`. Additionally, to note the port has changed 
+from 9000 to 9003
+
+```
+services:
+    cli:
+        environment:
+            - XDEBUG_CONFIG=remote_connect_back=0 remote_host=${DOCKSAL_HOST_IP}
+        expose:
```

Comment:
@lmakarov @achekulaev just checking real quick this is valid correct? As we are changing the port in the dockerfile to 9003 the old versions require 9000.

Reply:
✅